garden/sdf_worlds.md

```diff
@@ -76,44 +76,40 @@ Now let's define the GUI. Under the `<gui>` tag we specify anything related to t
 
 [gazebo-gui](https://github.com/gazebosim/gz-gui/) has a bunch of plugins to choose from. We will add the ones that are necessary to get our world up and running with basic functionality.
 
+### Scene 3D plugin
+
 ```xml
 <!-- 3D scene -->
-<plugin filename="MinimalScene" name="3D View">
-  <gz-gui>
-    <title>3D View</title>
-    <property type="bool" key="showTitleBar">false</property>
-    <property type="string" key="state">docked</property>
-  </gz-gui>
-
-  <engine>ogre2</engine>
-  <scene>scene</scene>
-  <ambient_light>0.4 0.4 0.4</ambient_light>
-  <background_color>0.8 0.8 0.8</background_color>
-  <camera_pose>-6 0 6 0 0.5 0</camera_pose>
-  <camera_clip>
-    <near>0.25</near>
-    <far>25000</far>
-  </camera_clip>
-</plugin>
-<plugin filename="GzSceneManager" name="Scene Manager">
-  <gz-gui>
-    <property key="resizable" type="bool">false</property>
-    <property key="width" type="double">5</property>
-    <property key="height" type="double">5</property>
-    <property key="state" type="string">floating</property>
-    <property key="showTitleBar" type="bool">false</property>
-  </gz-gui>
+<plugin filename="MinimalScene">
+    <gz-gui>
+      <title>3D View</title>
+      <property type="string" key="state">docked</property>
+    </gz-gui>
+    <engine>ogre2</engine>
+    <scene>scene</scene>
+    <ambient_light>0.4 0.4 0.4</ambient_light>
+    <background_color>0.8 0.8 0.8</background_color>
+    <camera_pose>-6 0 6 0 0.5 0</camera_pose>
 </plugin>
 ```
 
-`MinimalScene` and `GzSceneManager` are responsible for displaying the 3D scene of our world.
-It has the following properties (most of the GUI plugins have them):
+The `MinimalScene` plugin is responsible for displaying the 3D scene of our world. It has the following properties (most of the GUI plugins have them):
```

Comment:
Should we keep `GzSceneManager` here since nothing would work without it? cc @ahcorde